src/main/resources/instructorEmailFragment-courseJoin.html

```diff
@@ -11,4 +11,9 @@
     have a Google account, please create one from the 
     <a href="https://accounts.google.com/NewAccount">Google Accounts page</a>.
     <br>*The above link is unique to you. Please do not share it with others.
+</p>
+
+<p>
+    <strong>You were invited by ${inviterName}.</strong> 
+    You can contact him/her by following email: ${inviterEmail}
```

Comment:
Suggest putting this info at the end of the first paragraph instead, something like "You have been assigned an instructor role for the course by ${inviterName} who can be reached at ${inviterEmail}."